.gitignore

```diff
@@ -43,6 +43,24 @@ ExportedObj/
 *.booproj
 *.svd
 
+# Binary windows
+*.dll
```

Comment:
Not really keen on gitignoring dll libs as there is a possibility of using appropriately licensed libs of that type in the assets. Everything else is fine